src/components/Join/index.css

```
@@ -0,0 +1,78 @@
+.join {
+  inline-size: 1104px;
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  margin-top: 200px;
+}
+
+.join form {
+  inline-size: 100%;
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+}
+
+.join input {
+  inline-size: 300px;
+  padding: 10px 10px 10px 10px;
```

네 면의 값이 모두 같은 경우 단축해서 쓸 수도 있습니다. 혹시 이후 변경될 경우를 대비해 의도적으로 작성해둔 것이라면 무시해도 무방한 코멘트입니다😉
```suggestion
  padding: 10px;
```

https://developer.mozilla.org/ko/docs/Web/CSS/padding
한 개의 값은 모든 네 면의 여백을 설정합니다.

---

수정 완료했습니다 🙇‍♀️🙇‍♂️